src/test/java/lotto/domain/LottoTicketTest.java

```diff
@@ -11,14 +10,14 @@ public class LottoTicketTest {
 
     @Test
     void 로또_번호_6개_발급() {
-        LottoTicket numbers = new LottoTicket(RandomNumberUtils.generateRandomNumber());
+        NumberGenerator numberGenerator = new LottoNumberGenerator();
```

Comment:
`NumberGenerator` 를 인터페이스로 분리했으니 그 장점을 활용해볼까요?
테스트용으로만 활용할 `NumberGenerator` 의 구현체를 만드는 거에요.

```java
class TestNumberGenerator implement NumberGenerator {
    @Override
    public List<Integer> generate() {
        return Arrays.asList(1,2,3,4,5,6);
    }
}
```

위에서 `NumberGenerator` 가 `@FunctionalInterface`의 장점을 취했으면 좋겠다는 다른 리뷰를 남겼는데요,
만약 `NumberGenerator`의 메소드가 하나면 이렇게 람다를 활용할 수도 있습니다.
```suggestion
        NumberGenerator numberGenerator = () -> Arrays.asList(1,2,3,4,5,6);
```
그럼 원하는 번호(1, 2, 3, 4, 5, 6)가 바로 주입이 되는거죠!

Reply:
코멘트 내용을 참조하여 TestNumberGenerator 와 람다를 활용한 로직 모두 적용해보았습니다
이렇게 테스트가 간편해지는 장점이 있다니... 로직으로 직접 배울 수 있었습니다 👍 😆